src/routes/tokens/ApiTokens.tsx

```diff
@@ -0,0 +1,299 @@
+                                disabled={incorrectExpiration}
+                            >
+                                Generate Token
+                            </Button>
+                        </Tooltip>
+                    </ModalFooter>
+                </ModalContent>
+            </Modal>
+            {generatedToken && (
+                <Box>
+                    <Alert
+                        status="info"
+                        overflow="visible"
+                        background="green.100"
+                        padding={3}
+                    >
+                        <AlertIcon as={RiInformationLine} h={8} color="black" />
+
+                        <Box
+                            alignItems="stretch"
+                            width="100%"
+                            padding={1}
+                            background="green.100"
+                        >
+                            <AlertTitle fontSize="lg" color="black">
+                                Token generated! But it will be visible only
+                                once
+                            </AlertTitle>
+                            <AlertDescription
+                                maxWidth="sm"
+                                fontSize="sm"
+                                color="black"
+                            >
+                                Make sure to copy your API token now as you will
+                                not be able to see it again
+                            </AlertDescription>
+                            <HStack
+                                alignItems="space-between"
+                                spacing="0"
+                                width="45%"
+                            >
+                                <Text
+                                    background="white"
+                                    padding="2"
+                                    fontSize="sm"
+                                    color="black"
+                                >
+                                    {' '}
+                                    {generatedToken.token}
+                                </Text>
+                                <CopyIconButton
+                                    value={generatedToken.token}
+                                    aria-label="copy token"
+                                />
+                            </HStack>
+                        </Box>
+                    </Alert>
+                    <ApiToken key={generatedToken.id} token={generatedToken} />
+                </Box>
+            )}
+            {activeTokens.length !== 0 ? (
+                activeTokens.map((token) => (
+                    <ApiToken key={token.id} token={token} />
+                ))
+            ) : (
+                <Center>
+                    <HStack spacing="2">
+                        <Text>You don't have any active token.</Text>
```

Comment:
You could argue against this, but IMO this should be a plural.

```suggestion
                        <Text>You don't have any active tokens.</Text>
```